bindings/python/examples/gyroid_module.py

```diff
@@ -0,0 +1,46 @@
+"""
+ Copyright 2023 The Manifold Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ """
+
+import math
+import numpy as np
+from manifold3d import Mesh, Manifold
+
+def gyroid(x, y, z):
+    xi = x - math.pi / 4.0
+    yi = y - math.pi / 4.0
+    zi = z - math.pi / 4.0
+    return (math.cos(xi) * math.sin(yi) +
+            math.cos(yi) * math.sin(zi) +
+            math.cos(zi) * math.sin(xi))
+
+def gyroid_levelset(level, period, size, n):
+    return Manifold.from_mesh(
+       Mesh.level_set(gyroid, 
+                      [-period, -period, -period, period, period, period], 
+                      period / n, level)).scale(size / period)
+
+def rhombic_dodecahedron(size):
+  box = Manifold.cube(size * math.sqrt(2.0) * np.array([1, 1, 2]), True)
```

Comment:
maybe need to format this file to make sure the indentation is consistent.

Reply:
Good spot, I regularized the indentation.